paymentsheet/src/test/java/com/stripe/android/utils/RecordingLinkStore.kt

```diff
@@ -0,0 +1,37 @@
+package com.stripe.android.utils
+
+import app.cash.turbine.ReceiveTurbine
+import app.cash.turbine.Turbine
+import com.stripe.android.link.account.LinkStore
+import org.mockito.kotlin.doAnswer
+import org.mockito.kotlin.mock
+
+object RecordingLinkStore {
+    fun noOp(): LinkStore {
+        return mock()
+    }
+
+    suspend fun test(test: suspend Scenario.() -> Unit) {
+        val markAsUsedCalls = Turbine<Unit>()
+
+        val linkStore = mock<LinkStore> {
+            on { markLinkAsUsed() } doAnswer {
+                markAsUsedCalls.add(Unit)
+            }
+        }
```

Comment:
Same for `LinkStore`, a final class. Can be replaced by a test implementation if we refactor.